bootcfg/http/ignition.go

```diff
@@ -39,6 +37,20 @@ func ignitionHandler(srv server.Server) ContextHandler {
 			return
 		}
 
+		// Skip rendering if raw Ignition JSON is provided
+		if isIgnition(profile.IgnitionId) {
+			cfg, err := ignition.Parse([]byte(contents))
```

Comment:
The (rare) user who decides to supply raw Ignition JSON must write an Ignition config whose version is equal to or less than the Ignition.Parse vendored with that release of bootcfg.